setup.py

```diff
@@ -120,6 +120,14 @@ def read(fname):
         package_data={
             '': ['*.txt', '*.rst', '*.plug', '*.md'],
         },
+        extras_require={
+            'graphic':  ['PySide', ],
+            'hipchat': ['hypchat', 'sleekxmpp', ],
```

Comment:
sleekxmpp needs the `pyasn1` and `pyasn1-modules` in order to do certificate validations but doesn't have a hard dependency on them, so we should add those too.